v2/errors.go

```diff
@@ -38,7 +38,17 @@ type HTTPStatusCodeError struct {
 }
 
 func (e HTTPStatusCodeError) Error() string {
-	return fmt.Sprintf("Status: %v; ErrorMessage: %v; Description: %v; ResponseError: %v", e.StatusCode, e.ErrorMessage, e.Description, e.ResponseError)
+	if e.ErrorMessage == nil {
```

Comment:
So we can dereference and show the contents in the error message. Please help me understand if I'm mistaken. I've tried to illustrate the difference I think applies here: https://goplay.space/#l4h_sMpM9q